pkg/blob/store.go

```diff
@@ -62,8 +62,10 @@ type Store interface {
 	BucketName() string
 	Copy(batch CopyBatch)
 	CopyOne(obj *CopyObject) error
+	CreateBucket()
```

Comment:
This is not required. The bucket is created as part of NewStore.

Reply:
Required to be used by the writer. Also the writer needs to create a bucket after purge (if purge enabled).